supertokens_python/recipe/session/session_class.py

```diff
@@ -112,3 +119,77 @@ async def get_expiry(self, user_context: Union[Dict[str, Any], None] = None) ->
             raise_unauthorised_exception("Session does not exist anymore.")
 
         return session_info.expiry
+
+    async def assert_claims(
+        self,
+        claim_validators: List[SessionClaimValidator],
+        user_context: Union[Dict[str, Any], None] = None,
+    ) -> None:
+        if user_context is None:
+            user_context = {}
+
+        original_session_claim_payload_json = json.dumps(
+            self.get_access_token_payload()
+        )
+
+        new_access_token_payload = await update_claims_in_payload_if_needed(
+            claim_validators,
+            self.get_access_token_payload(),
+            self.get_user_id(),
+            user_context,
+        )
+
+        if json.dumps(new_access_token_payload) != original_session_claim_payload_json:
+            await self.merge_into_access_token_payload(
+                new_access_token_payload, user_context
+            )
+
+        validation_errors = await validate_claims_in_payload(
+            claim_validators, new_access_token_payload, user_context
+        )
+
+        if len(validation_errors) > 0:
+            raise_invalid_claims_exception("INVALID_CLAIMS", validation_errors)
+
+    async def fetch_and_set_claim(
+        self, claim: SessionClaim[Any], user_context: Union[Dict[str, Any], None] = None
+    ) -> None:
+        update = await claim.build(self.get_user_id(), user_context)
+        return await self.merge_into_access_token_payload(update, user_context)
+
+    async def set_claim_value(
+        self,
+        claim: SessionClaim[_T],
+        value: _T,
+        user_context: Union[Dict[str, Any], None] = None,
+    ) -> None:
+        update = claim.add_to_payload_({}, value, user_context)
+        return await self.merge_into_access_token_payload(update, user_context)
+
+    async def get_claim_value(
+        self, claim: SessionClaim[Any], user_context: Union[Dict[str, Any], None] = None
+    ) -> Union[Any, None]:
+        return claim.get_value_from_payload(
+            self.get_access_token_payload(), user_context
```

Comment:
Pass user context to get_access_token_payload

Reply:
done